### DISCUSSION TITLE
NOOB has a question

### DISCUSSION THREAD

**Discussion by ghost** [waylaidwanderer/node-chatgpt-api#332] [Discussions]
How does it work? At first glance, it was a mock web version sending packets to the server. Is that the principle?

**Comment by waylaidwanderer** [comment]
Which client are you referring to? You can look through the code for each client in the [`src` folder](https://github.com/waylaidwanderer/node-chatgpt-api/tree/main/src).

If there's something you don't understand, ask ChatGPT to explain it to you :)
